pkg/controllers/user/rbac/namespace_handler.go

```
@@ -118,6 +120,30 @@ func (n *nsLifecycle) ensurePRTBAddToNamespace(ns *v1.Namespace) (bool, error) {
 			return false, errors.Wrapf(err, "couldn't ensure binding %v in %v", prtb.Name, ns.Name)
 		}
 	}
+
+	prtbSet := map[string]string{}
+	prtbList := n.m.prtbIndexer.List()
+	for _, prtb := range prtbList {
+		prtb, ok := prtb.(*v3.ProjectRoleTemplateBinding)
+		if !ok {
+			return false, err
+		}
+		prtbSet[convert.ToString(prtb.GetUID())] = prtb.Namespace
+	}
+	rbs, _ := n.m.workload.K8sClient.RbacV1().RoleBindings(ns.Name).List(metav1.ListOptions{})
```

you cant just swallow this error